packages/react-components/react-datepicker-compat/src/components/DatePicker/DatePicker.types.ts

```diff
@@ -3,6 +3,7 @@ import { Input } from '@fluentui/react-input';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 import type { CalendarProps } from '../Calendar/Calendar.types';
 import type { CalendarStrings, DateFormatting } from '../../utils';
+import type { PortalProps } from '@fluentui/react-portal';
```

Comment:
You probably need to import `PortalState` now as well

```suggestion
import type { PortalProps, PortalState } from '@fluentui/react-portal';
```